### PR TITLE
doc comments for algo.cmp.among

### PR DESCRIPTION
**Pull request by burner**
Params und Returns

### CODE REVIEW

File: std/algorithm/comparison.d [hunk 1]
```diff
@@ -63,6 +63,15 @@ of the first matching value in $(D values), or $(D 0) if $(D value)
 is not _among $(D values). The predicate $(D pred) is used to
 compare values, and uses equality by default.
 
+Params:
+    pred = The predicate used to compare the values.
+    Value = The value to search.
```
Comment (JakobOvrum) [inline std/algorithm/comparison.d]: s/Value/value/

File: std/algorithm/comparison.d [hunk 2]
```diff
@@ -63,6 +63,15 @@ of the first matching value in $(D values), or $(D 0) if $(D value)
...
+    Values = The values to compare the Value to.
```
Comment (JakobOvrum) [inline std/algorithm/comparison.d]: s/Values/values/

**Comment by burner** [comment]
I actually believe it should all be lowercase

File: std/algorithm/comparison.d [hunk 3]
```diff
@@ -63,6 +63,15 @@ of the first matching value in $(D values), or $(D 0) if $(D value)
 is not _among $(D values). The predicate $(D pred) is used to
 compare values, and uses equality by default.
 
+Params:
+    pred = The predicate used to compare the values.
+    value = The value to search.
```
Comment (JakobOvrum) [inline std/algorithm/comparison.d]: Nitpick, but this wording technically means that something is searched for in *value*. Should be "The value to search for."

Comment (burner) [inline std/algorithm/comparison.d]: thanks

**Comment by JakobOvrum** [comment]
Auto-merge toggled on

**Comment by burner** [comment]
thanks

**Pull request merged**